### PR TITLE
test: fix more validation issues

### PR DESCRIPTION
**Pull request by larseggert**
Including the ones @rjsparks sent me in email

### CODE REVIEW

**Comment by larseggert** [comment]
Not sure why the tests fail, they succeed locally

File: ietf/nomcom/views.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def index(request):
         else:
             nomcom.url = None
         if year >= 2002:
-            nomcom.ann_url = "/nomcom/ann/#%4d" % year
+            nomcom.ann_url = "/nomcom/ann/#nomcom-%4d" % year
```
Comment (rjsparks) [inline ietf/nomcom/views.py:60]: We'll go with this for now, but consider dropping the hyphen since nomcom2021 is a group.

**Review by rjsparks — APPROVED** [review]

File: ietf/static/css/liaisons.css [hunk 2]
```diff
@@ -1,67 +1,4 @@
...
-.baseform #id_organization,
-.baseform #id_to_poc,
-.baseform #id_response_contacts,
-.baseform #id_technical_contact,
-.baseform #id_cc1,
-.attach_titleField input,
-.baseform textarea {
-    width: 80%;
-}
+/* TODO: this should use bs5 styling and ideally be integrated into the code */
```
Comment (rjsparks) [inline ietf/static/css/liaisons.css:1]: What tooling are you using that shows the removed styling was not used?

Comment (larseggert) [reply ietf/static/css/liaisons.css:1]: I grepped for "baseform" in the repo :-)

**Pull request merged**